apps/data/src/pipelines/centrum_pipeline.py

```
@@ -782,6 +784,135 @@ def execute_macro_udf(pdf: pd.DataFrame) -> pd.DataFrame:
+            F.col("macro.name").alias("macro_name"),
+            F.col("macro.filename").alias("macro_filename"),
+        )
+    )
+
+    return (
+        base_df
+        # Run sandbox UDF only for non-imported rows
+        .withColumn(
+            "sandbox_result",
+            F.when(
+                ~F.coalesce(F.col("skip_macro_processing"), F.lit(False)),
```

Ruff (0.15.10)

[warning] Boolean positional value in function call

(FBT003)